### ISSUE TITLE
Log upload: Data Integrity Issue

### ISSUE THREAD

**Issue by gdude2002** [PrismLauncher/PrismLauncher#930]
### Operating System

Windows 11

### Version of Prism Launcher

Prism Launcher 5.2

### Version of Qt

Qt 6.4.0 (Windows, so this must be bundled)

### Description of bug

When logs are uploaded to mclo.gs from within Prism, it seems that the data uploaded isn't entirely correct - in my case, this results in an extra line of broken data within Quilt Loader's mods table. Below, you can see a screenshot of VS Code with word wrap disabled, showing the difference between three methods of viewing the file - directly on mclo.gs as uploaded by Prism, directly in the file on disk, and within Prism's own log viewer.

This does not happen if I upload the file to mclo.gs myself.

![image](https://user-images.githubusercontent.com/204153/224735637-9c345afb-a28d-4044-af27-b6d59e2e82f8.png)

* [The log file I'm working with](https://github.com/PrismLauncher/PrismLauncher/files/10958508/latest.log)
* [The log file Prism uploaded](https://mclo.gs/xdcIjE6)

This problem breaks Cozy's new log parser, as the data is broken in a way that my parser can't reasonably account for.

### Steps to reproduce

* Start a modpack making use of a recent version of Quilt Loader, and a large number of mods
* Close the game and upload the logs via Prism's "upload logs" button
* Compare the mods table in the logs that were uploaded to mclo.gs with the table in the latest.log file and in Prism's own log viewer

### Suspected cause

When copying logs from Prism's viewer into VS Code, it complains that Prism is adding strange line endings. It may be worth double-checking how Prism is sanitizing the line endings in the logs, if it is doing so at all.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by Scrumplex** [comment]
There was some bug with line breaks in Qt 6.4.0 IIRC.

Let me find the relevant issue.

**Comment by Scrumplex** [comment]
Here it is: #288

**Comment by gdude2002** [comment]
OK, that explains my copy-paste issue, but it doesn't explain why the uploaded log is slightly corrupted.

I've updated the original comment with the logfile Prism uploaded as well.

**Comment by Scrumplex** [comment]
This is the code that triggers the log upload: https://github.com/PrismLauncher/PrismLauncher/blob/abc2a529fb8ff63af6e5f9ab5443486be9e40f2f/launcher/ui/pages/instance/LogPage.cpp#L285

https://github.com/PrismLauncher/PrismLauncher/blob/abc2a529fb8ff63af6e5f9ab5443486be9e40f2f/launcher/launch/LogModel.cpp#L85-L96

Hmm. Can you check if this issue persists with Prism Launcher 6.3?

**Comment by gdude2002** [comment]
> Can you check if this issue persists with Prism Launcher 6.3?

No, I can't.

![image](https://user-images.githubusercontent.com/204153/224743583-fee06ef7-a4d8-456c-b2a6-78aa17b5ae23.png)

**Comment by Scrumplex** [comment]
That's #660 

TL;DR: Do a full re-install of Prism (if you used the installer, uninstall and then install the new version)

**Comment by gdude2002** [comment]
I can confirm that [this still happens with 6.3](https://mclo.gs/YFZqLwf).

**Comment by Scrumplex** [comment]
I can reproduce this issue :thinking:

**Comment by gdude2002** [comment]
Looking forward to hearing about a release!
